.gitignore

```diff
@@ -123,6 +123,19 @@ fabric.properties
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
 
+### JDK ###
+.idea
```

Comment:
```suggestion
```
This is handled above as well (IntelliJ)

Reply:
the idea tags above are intellij specific

Reply:
and won't get rid of certain files

Reply:
If you want to just ignore .idea/, I suggest removing the above redundant lines that are more specific e.g. 5-41, 49, 64, 92, etc.